test/Dapr.Actors.Test/DaprFormatTimeSpanTests.cs

```diff
@@ -3,6 +3,7 @@
 // Licensed under the MIT License.
 // ------------------------------------------------------------
 
+#pragma warning disable 0618
```

Comment:
`TimerInfo` is obsolete. These are dotted around. When we do a major release we'll make it internal and clean this up.